src/fb5.js

```diff
@@ -136,14 +142,12 @@ function setWatchObserver() {
 function onPageChange() {
   if (isFBWatch()) {
     if (feedObserver !== null) {
```

Comment:
We don't need this check here anymore right? Since we have the same check inside `cleanupFeedObserver`?

Reply:
changed